bpf_test.go

```diff
@@ -234,6 +239,26 @@ func checkSocketFilters(t *testing.T, b *elf.Module) {
 	}
 }
 
+func checkPinCustomConfig(t *testing.T, b *elf.Module) {
+	// NOTE: this test doesn't really work kernel 4.9 or older,
+	// because /sys/fs/bpf/fw doesn't exist at all.
```

Comment:
@alban I'm wondering too which one exactly causes the failure. Just looking at the wiki, it isn't clear to me.